### PR TITLE
Update NEWS.md and README.md

### PR DESCRIPTION
**Pull request by iguessthislldo**

### CODE REVIEW

**Review by mitza-oci — APPROVED** [review]

File: README.md
Comment (mitza-oci) [inline README.md:105]:
```suggestion
* DOC Group ACE 6.5.13 / TAO 2.5.13 or later in the ACE 6.x / TAO 2.x series
```

**Review by mitza-oci — CHANGES_REQUESTED** [review]

File: README.md
Comment (jwillemsen) [inline README.md:105]: Shouldn't this be extended with ACE 7.0.2 / TAO 3.0.2?

Comment (mitza-oci) [reply README.md:105]: There isn't the level of testing across multiple platforms on ACE 7 that would lead us to call it "supported."  It's good that we're minimally testing it in CI so we don't accidentally introduce some incompatibility, but this is essentially for maintainers and not for end users.

**Review by mitza-oci — APPROVED** [review]

File: README.md
Comment (mitza-oci) [inline README.md:109]:
```suggestion
    * `--ace-github-latest` to use the ace6tao2 branch of ACE/TAO as is. This
```

**Review by mitza-oci — CHANGES_REQUESTED** [review]
Also holding this open so we can add late-breaking changes to the NEWS

**Review by mitza-oci — APPROVED** [review]

**Pull request merged**
